gamification/src/Components/TeacherGroups/TeacherGroups.js

```diff
@@ -20,8 +20,8 @@ function TeacherGroups() {
         </Row>
         <Row>
           <Col>
-            <p>Cards con los grupos - incluyen botones de editar y borrar (?)</p>
```

Comment:
add tables with subjects todo for @FanchiniRudolf

Reply:
Component name changed to Subjects.js
Todo added